docs/source/build_instructions.md

```diff
@@ -62,18 +62,18 @@ Download the
 [depot_tools bundle](https://storage.googleapis.com/chrome-infra/depot_tools.zip)
 and extract it somewhere.
 
-*** note
+***Note***  
```

Comment:
Thanks for fixing this. It is certainly better than the original code, but I am not sure if it is what we want.

What do you think of just replacing this line with just "***"?